maps/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import "fmt"
+
+func main() {
+	m := make(map[string]int)
+	m["k1"] = 44
+	m["k2"] = 45
+	m["k3"] = 67
+	fmt.Println("Map", m)
+
+	fmt.Println("================================")
+
+	n := make(map[string]int)
+	n["key1"] = 45
+	n["key2"] = 40
+	n["key3"] = 48
+	n["key4"] = 60
+	fmt.Println("Map", n)
+
+	fmt.Println("================================")
+
+	name := make(map[string]int)
+	name["Akash"] = 56
+	name["Atharva"] = 57
+	name["Vivek"] = 67
+	name["Sushant"] = 98
+	fmt.Println("Names Are", name)
+
+	fmt.Println("================================")
+
+	v1 := m["k1"]  //Here The Value For key Is Fetched With "name[key]" i.e. k1
```

Comment:
What will happen if the key is something that is not used before?

Reply:
For That We Need To Declare The Variable And Assign Value To It And FOr The Same Key Must Be Assigned If it's Not Done It Will Give An Error Undeclared Name.

Reply:
And Also It Will Not Give An Output It Will Skip That Line To Execute If We Provide The Key Which Is Not Used

Reply:
@atharvaskadam - `And Also It Will Not Give An Output It Will Skip That Line To Execute If We Provide The Key Which Is Not Used` 

Output is simply not visible, and it doesn't skip it, it simply assigns the default value for that type, which in your case the class is `int` so the value will become `0`. Try it once, you will understand.

Reply:
Noted @aasisodiya I'll Try It